notebooks/206-vision-paddlegan-anime/206-vision-paddlegan-anime.ipynb

```diff
@@ -0,0 +1,614 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "7607ce35-db52-4e1c-add7-8abed748de6a",
+   "metadata": {},
+   "source": [
+    "# Photo to Anime with PaddleGAN and OpenVINO\n",
+    "\n",
+    "This notebook demonstrates converting a [PaddlePaddle/PaddleGAN](https://github.com/PaddlePaddle/PaddleGAN) AnimeGAN model to OpenVINO IR format, and shows inference results on the PaddleGAN and IR models.\n",
+    "\n",
+    "https://github.com/PaddlePaddle/PaddleGAN/blob/develop/docs/en_US/tutorials/animegan.md"
```

Comment:
```suggestion
    "https://github.com/PaddlePaddle/PaddleGAN/blob/develop/docs/en_US/tutorials/animegan.md\n",
    "\n",
    "<img src=\"https://user-images.githubusercontent.com/15709723/123559130-04550100-d74f-11eb-819c-a02284654428.jpg\">"
```

Reply:
I suggest adding a before and after image to the top, to visually show what the model does